vagrant/provision.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Install prerequisite packages
+apt-get update
```

Comment:
Does this happen only once, or on every reload/launch of the VM?

Reply:
The provision script only runs on the initial spin-up of the VM.
The build script runs every time the VM gets reloaded or a user types `vagrant up`.

Examples:
`vagrant up` (very first time) - build and provision scripts run
`vagrant up` (every time after the first time) - build script runs
`vagrant reload` - build script runs
`vagrant reload --provision` - build and provision scripts run